src/functions.py

```diff
@@ -9,4 +9,9 @@
 # Print out "Even!" if the number is even. Otherwise print "Odd"
 
 # YOUR CODE HERE
-
+def is_even(n):
```

Comment:
Minor issue: spec mentioned is_even() is returning a boolean value, so try to utilize it to set your conditional print statements outside the method.